src/atc/delta/delta_handle.py

```diff
@@ -99,6 +103,88 @@ def recreate_hive_table(self):
         self.drop()
         self.create_hive_table()
 
-    def merge(self):
-        # This should be a method which can merge from delta to delta
-        raise NotImplementedError
+    def get_tablename(self) -> str:
+        return self._name
+
+    def upsert(
+        self,
+        df: DataFrame,
+        join_cols: List[str],
+        incremental_load: bool = True,
```

Comment:
Why have the parameter incremental_load? As i se it, incremental_load=False result in a write_or_append(mode="overwrite"), so why not just call that from the user.

Reply:
Agree. If one needs to make fullload, one should use write_or_append(mode="overwrite").